hardhat.config.js

```diff
@@ -74,5 +74,21 @@ module.exports = {
         mirrorNode: NETWORKS.previewnet.mirrorNode,
       },
     },
+     // besu local node
+     besu_local: {
```

Comment:
Ooh we're running a local besu client now? Is this a must or something? I think this is something we need to create a ticket for.

Reply:
We are only running it for testing.  When we need to compare behaviour between our and their systems.
Nothing in production is running on besu.
I left the config because it's useful